src/main/java/duke/task/EventObj.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * @author leslieharland
+ */
+
+package duke.task;
+
+/**
+ * The Class EventObj.
+ */
+public class EventObj extends Task {
+
+    /**
+     * Instantiates a new event obj.
+     *
+     * @param description the description
+     * @param datetime    the datetime
+     */
+    public EventObj(String description, String datetime) {
+        super(description, datetime);
+        type = "E";
+    }
+
+    /**
+     * Parses the event.
+     *
+     * @param taskString the task string
+     * @return the event obj
+     */
+    public static EventObj parse(String taskString) {
+        String[] parts = taskString.split("\\|");
+        String desc = parts[2];
+        EventObj t = new EventObj(desc, "");
+        if (parts.length > 3) {
+            t = new EventObj(desc,
+                    String.join("", java.util.Arrays.stream(parts).skip(3).toArray(String[]::new)));
+        }
+        t.isDone = parts[1].trim().equals("1");
+
+        return t;
+    }
+
+    /**
+     * To string.
```

Comment:
It would be better if the java comment could explain how the toString method formats the event string.